src/items.rs

```diff
@@ -3399,28 +3420,9 @@ impl Rewrite for ast::ForeignItem {
                     .map(|(s, _, _)| format!("{};", s))
                 }
             }
-            ast::ForeignItemKind::Static(ref static_foreign_item) => {
-                // FIXME(#21): we're dropping potential comments in between the
-                // function kw here.
```

Comment:
To be honest I wasn't sure if this FIXME comment was misplaced or not. Would it be better to move this to the `ast::ForeignItemKind::Fn` match arm above?